### ISSUE TITLE
[Bug] Formatter breaks Customizer annotations for enumerables

### ISSUE THREAD

**Issue by Fordi** [alufers/openscad-parser#15]
The formatter inserts a line break after a declaration's semicolon.  If there's a customizer annotation after the declaration, that breaks the customizer's understanding of the annotation.

[Spec](https://en.wikibooks.org/wiki/OpenSCAD_User_Manual/Customizer#Syntax_support_for_generation_of_the_customization_form)

For example:

```javascript
const { CodeFile, ParsingHelper, ASTPrinter, FormattingConfiguration } = require('openscad-parser');
const [ast, errorCollector] = ParsingHelper.parseFile(new CodeFile('<test>', [
  '// An enumerable',
  'enum_value="a"; // [a:ayyyy, b:beee, c:seeee]',
].join('\n')));
errorCollector.throwIfAny();
console.log(new ASTPrinter(new FormattingConfiguration()).visitScadFile(ast));
```

Expected:
```openscad
// An enumerable
enum_value="a"; // [a:ayyyy, b:beee, c:seeee]
```

Actual:
```openscad
// An enumerable
enum_value = "a";
// [a:ayyyy, b:beee, c:seeee]
```

**Comment by Fordi** [comment]
Not urgent; I have a post-processing workaround for my stuff that's just a little dumb regexp.  Not ideal, but it'll work for now.

**Comment by alufers** [comment]
Hi! I will take a look at this and fix it. Thanks for using my project, didn't really expect anybody to find it :)

**Comment by alufers** [comment]
I have fixed this issue in `0.2.1`. I have also added `scadfmt` to the bin section of package json so now you can use it like this:

```sh
npm i -g openscad-parser # install
scadfmt <file> # use
```
